README.md

```diff
@@ -0,0 +1,65 @@
+# Cloudblue Connect Mulesoft Extension
+
+## Overview
+CloudBlue Connect is a supply automation platform that manages your products and services, contracts, ordering and fulfillment, usage and subscriptions.
```

Comment:
CloudBlue Connect is a supply automation platform that between others manages products, services, contracts, ordering and fulfillment, usage and subscriptions.

Reply:
Done